operators/src/source/gdal_source.rs

```diff
@@ -405,6 +454,13 @@ where
             .as_ref()
             .map(|o| o.iter().map(String::as_str).collect::<Vec<_>>());
 
+        // reverts the thread local configs on drop
+        info!("{:?}", dataset_params.gdal_config_options);
```

Comment:
Don't you `info!` right within calling `new`? Isn't it doubled?

Reply:
6aedc4ecc6cbc634b02eb93affae62211749444c